domains/transfer/Transfer.API/API/Claiming/Api/v2023_01_01/Controllers/ClaimAutomationController.cs

```diff
@@ -29,16 +29,18 @@ public ClaimAutomationController(IClaimAutomationRepository claimAutomationRepos
     [ProducesResponseType(typeof(ClaimAutomationArgumentDto), 200)]
     public async Task<ActionResult> StartClaimAutomation()
```

Comment:
Did you test that stuff still works in the preview environment?

Reply:
Yes, seems to work fine:

https://eo-u-userdescriptor.westeurope.cloudapp.azure.com/